EXT_MD.md

```diff
@@ -0,0 +1,132 @@
+      - `pair_coeff * * 1 1`
+      - `pair_coeff 1 1 Cu_mishin1.eam.alloy Cu`
+      - ...
+
+Observable Records
+---------------------------
+
+`observables` is an *optional* group that contains physical observables that are derived from the system state, i.e., thermodynamic information (e.g. temperature, energy, pressure).
+
+### Naming conventions
+
+The naming conventions conforming the naming conventions for [Scalar, Vector and Tensor Records](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md#naming-conventions) in [openPMD base standard](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md).
+
+- `scalar` record
+  - type: *(any type)*
+  - data set: `recordName` unique name in group `basePath` + `observables`
+  - examples:
+    - /data/observables/temperature
+    - /data/observables/pressure
+
+### Attributes for each `observable` record
+
+The attributes of unit system for records should be included as defined in [openPMD base standard](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md#unit-systems-and-dimensionality).
+
+
+Particle Records
+---------------------------
+
+### Additional attributes for each `position` record
+
+The following attributes are defined in this extension. The individual requirement is given in `scope`. 
+
+- `coordinate`
+    - type: *(string)*
+    - scope: *required*
+    - description: the representation of the stored position coordinates
+    - available values:
+      - `absolute` the unscaled coordinates
+      - `fractional` the coordinates that are scaled in the range of [0,1] relative to the length of each box edge; in this coordinate system, the `unitSI` of each position component should be `1.0`
+
+### Additional Sub-Group for each Particle Species
+
+`box` is an *optional* sub-group for each particle species to contain the information of the simulation box. 
+
+### Attributes for Sub-Group `box`
+
+The following attributes are defined in this extension.
+If the `box` sub-group exists, the following attributes within it are required.
+The individual requirement is given in `scope`. 
+
+  - `dimension`
+    - type: *(uint32)*
+    - scope: *required*, if `box` sub-group exists
+    - description: the spacial dimension **D** of the simulation box.
+    - example values:
+      - `2` 2D simulation box
+      - `3` 3D simulation box
+      - ...
+
+  - `boundary`
+    - type: array of *(string)* containing **D** elements, where **D** is the value of `dimension`.
+    - scope: *required*, if `box` sub-group exists
+    - description: the boundary condition of the box along each dimension. The allowed values in `boundary` are either **periodic** or **none**.
+    - example values:
+      - `["periodic","periodic","periodic"]`  periodic in all the three dimensions
+      - `["none","periodic","periodic"]` periodic in only *y* and *z* dimensions
+
+### Records for Sub-Group `box`
+
+The following records are defined in this extension.
```

Comment:
Do you want to store these small data sets as `records` or as `attributes`? Both is fine, just be aware that attributes can only be scalars or 1D-arrays, due to limitations in some file formats.

Reply:
Yes, I would like to use `records` here.